### DISCUSSION TITLE
Welcome to Amazing-Python-Scripts Discussions!

### DISCUSSION THREAD

**Discussion by avinashkranjan** [avinashkranjan/Amazing-Python-Scripts#154] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by divya-gh** [comment]
Hi, I am a data science student working on Data Analytics, ETL using python, pandas, SQL.  I'm new to open source. I would love to contribute to your projects in any way I can. Can you please guide me through it?

**Reply by Kushal997-das** [reply]
Yes @divya-gh We can help you :)
